### DISCUSSION TITLE
Filters

### DISCUSSION THREAD

**Discussion by ghost** [lnreader/lnreader#333] [Ideas]
Hello , I have a suggestion, what about adding filters in the app , for example in tachiyomi, when I open an extension the app adopts the website's filters and like that i'm able to find books suitable to my tastes faster than going to Web view mode , searching for stories then going back to app , it would be a great addition and thank you so much for your amazing work!

**Comment by rajarsheechatterjee** [comment]
Filters have already been added in the app. Currently, there are only some sources (NovelUpdates, ReadLightNovel & Russian sources) which have them as filters need to be manually added for each source. You can try them in the [preview release](https://github.com/LNReader/lnreader-preview/releases/tag/v1.1.10.2).

**Reply by Anondesu** [reply]
@rajarsheechatterjee Hello, good day, is the filters interactive like in tachiyomi's? I tried to press one of the filters but it didn't work. I'm using novel updates extension. Would be nice if there's also search filters.

**Reply by rajarsheechatterjee** [reply]
> is the filters interactive like in tachiyomi's? I tried to press one of the filters but it didn't work.

Yes. Bottomsheet interactions don't work properly on some devices like Samsung so it could be due to that.

> Would be nice if there's also search filters.

Someone would have to manually add them.

**Reply by Anondesu** [reply]
>Yes. Bottomsheet interactions don't work properly on some devices like Samsung so it could be due to that.

Oh I see, probably realme is included to that some devices that don't work.

>Someone would have to manually add them.

Oh that's a shame, that's a lot of work to do.


Anyway thanks for answering, I love the app.

**Reply by ghost** [reply]
It worked , thank you so much !
